content/main/docs/serverless-workflows/troubleshooting.md

```diff
@@ -97,37 +97,4 @@ When deploying a workflow in a namespace other than the one where Sonataflow ser
 
 ### Problem: sonataflow-platform-data-index-service pods can't connect to the database on startup
 
-First, check if the PostgreSQL pod is still starting. If it is, allow some time for the pod to become fully operational before the DataIndex and JobService pods are ready.
-
-If the PostgreSQL pod is running but the issue persists, verify that the `sonataflow-infra` namespace has the required label `rhdh.redhat.com/workflow-namespace`. Missing this label can cause connectivity issues.
-
-**Solution:**
-
-Get the NetworkPolicy name from its [definition](https://github.com/parodos-dev/orchestrator-helm-operator/blob/03ddd07bcd8e2aaf9fb37cedb6604be2cf37b6f3/helm-charts/orchestrator/templates/network-policies.yaml#L5). or example, it may be `allow-rhdh-to-sonataflow-and-workflows`.
-
-View the details of the specified NetworkPolicy:
-
-```bash
-oc get networkpolicy allow-rhdh-to-sonataflow-and-workflows -n sonataflow-infra
-```
-
-In the output YAML, you should see something like:
-
-```yaml
-- namespaceSelector:
-    matchLabels:
-    # Allow any other namespace the has workflows deployed because this is where
-    # this namespace contains the sonataflow services
-    rhdh.redhat.com/workflow-namespace: ""
-```
-
-verify if that label is available in `sonataflow-infra` namespace:
-
-```bash
-oc get ns sonataflow-infra -o yaml
-```
-
-If the required label is missing, you can add it by running the following command:
-```bash
-oc label ns sonataflow-infra rhdh.redhat.com/workflow-namespace: ""
-```
\ No newline at end of file
+Check if the PostgreSQL pod is still starting. If it is, allow some time for the pod to become fully operational before the DataIndex and JobService pods are ready.
```

Comment:
I'd mention here another case, in which the postgresql server is installed on its own namespace, other than the one in which sonataflow-services are installed (e.g. not in the sonataflow-infra namespace), and on that postgresql namespace there is a networkpolicy that doesn't have ingress configured to allow incoming traffic from the sonataflow-service namespace.

Reply:
Done